app/frontend/app/controllers/kotodama.js

```diff
@@ -8,6 +8,8 @@ export default Ember.ObjectController.extend({
   blotterMesgFix: "",
   blotterMesg: "",
   blotterLink: "",
+  userToFollow: "",
+  userToFollowSubmit: false,
```

Comment:
Maybe this should be `userToFollowSubmitted` for more clarity, since it is a state variable?